dev-libs/atf/atf-0.21.recipe

```diff
@@ -0,0 +1,111 @@
+SUMMARY="Libraries to write tests in C, C++ and shell"
+DESCRIPTION="ATF, or Automated Testing Framework, is a collection of libraries \
+to write test programs in C, C++ and POSIX shell.
+
+The ATF libraries offer a simple API. The API is orthogonal through \
+the various bindings, allowing developers to quickly learn how to write \
+test programs in different languages.
+
+ATF-based test programs offer a consistent end-user command-line interface to \
+allow both humans and automation to run the tests.
+
+ATF-based test programs rely on an execution engine to be run and \
+this execution engine is not shipped with ATF. Kyua is the engine of choice."
+HOMEPAGE="https://github.com/jmmv/atf"
+COPYRIGHT="2007-2012 The NetBSD Foundation, Inc.
+	2011, 2012, 2014 Google Inc."
+LICENSE="BSD (2-clause)
+	BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/jmmv/atf/releases/download/atf-$portVersion/atf-$portVersion.tar.gz"
+CHECKSUM_SHA256="92bc64180135eea8fe84c91c9f894e678767764f6dbc8482021d4dde09857505"
+PATCHES="atf-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86_64"
```

Comment:
Shall we add `?x86` ?
```diff
-ARCHITECTURES="!x86_gcc2 x86_64"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
```

Reply:
Do we still support `x86` as a main architecture?

Reply:
Nope. x86 is only supported as a second arch for x86_gcc2 primary arch.

That said, as we know, if a package builds fine on x86 secondary arch then it will also build fine on x86 primary arch, unless the recipe uses hard coded paths or if it depends on recipes which do not have `x86` (or `?x86`) in `ARCHITECTURES`...

So, I think it would be nice to keep a `?x86` in `ARCHITECTURES` so that those users (like myself) who are still running Haiku x86 will be able to play with pkgconf.

BTW, I'm not against the switch to x86_64. Actually, I can run all "flavours" of Haiku at the same time using separate VMs of course. And they all use a common haikuports tree, shared via NFS.
My 3 Haiku systems are on (different partitions of) a single SDRAM which I can even use to boot real hardware.

So, if you agree with having a `?x86` in `ARCHITECTURES` that would be cool :)